<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cypherEditGridWidget</class>
 <widget class="QWidget" name="cypherEditGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frmEditnGrid">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QFrame" name="frmEdit">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QFrame" name="frmEditorButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QToolButton" name="btnRun">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Run Query at cursor</string>
               </property>
               <property name="text">
                <string>run</string>
               </property>
               <property name="icon">
                <iconset resource="../icons/icons.qrc">
                 <normaloff>:/icons/RUN and DATA TAB/Run.png</normaloff>:/icons/RUN and DATA TAB/Run.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnRunScript">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Run File as Script</string>
               </property>
               <property name="text">
                <string>script</string>
               </property>
               <property name="icon">
                <iconset resource="../icons/icons.qrc">
                 <normaloff>:/icons/RUN and DATA TAB/RunScript.png</normaloff>:/icons/RUN and DATA TAB/RunScript.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>30</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbAutoCommit">
               <property name="minimumSize">
                <size>
                 <width>121</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>121</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>How To Commit</string>
               </property>
               <item>
                <property name="text">
                 <string>Auto Commit On</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Auto Commit Off</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnCommit">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Commit Current Transaction</string>
               </property>
               <property name="text">
                <string>Commit</string>
               </property>
               <property name="icon">
                <iconset resource="../icons/icons.qrc">
                 <normaloff>:/icons/RUN and DATA TAB/Commit.png</normaloff>:/icons/RUN and DATA TAB/Commit.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnRollBack">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="toolTip">
                <string>RollBack Current Transaction</string>
               </property>
               <property name="text">
                <string>RollBack</string>
               </property>
               <property name="icon">
                <iconset resource="../icons/icons.qrc">
                 <normaloff>:/icons/RUN and DATA TAB/RollBack.png</normaloff>:/icons/RUN and DATA TAB/RollBack.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>175</width>
                 <height>19</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frmData">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QFrame" name="frmDataGrid">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
